addons/web/static/src/views/kanban/kanban_controller.scss

```diff
@@ -164,15 +155,15 @@
 
             .o_kanban_image_fill_left {
                 position: relative;
-                margin-right: $o-kanban-inside-hgutter*2;
+                margin-right: var(--Kanban__record-padding-h, #{$o-kanban-inside-hgutter});
                 @include media-breakpoint-up(sm) {
                     margin: {
```

Comment:
Debatable If nesting properties it's useful or not...
IMO it just adds complexity and, in case like these ones, leads to misinterpretations.
At first sight I may assume that `top`, `bottom` and `left` are defining the element position rather than its margin.

You can keep it if you want, just my 2cents ;)